railties/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Remove duplicated `features` attribute on devcontainer.json.tt file.
+
+    Prior to this commit we had the `features` attribute duplicated in the `devcontainer.json.tt` file.
+
+    *Cadu Ribeiro*
```

Comment:
The devcontainer hasn't been released yet as far as I can tell, so we probably don't need a new changelog entry. Can you remove it please?

Reply:
@carlosantoniodasilva makes sense! 

I removed the changelog entry